src/freeze-dry/fix-links.test.js

```diff
@@ -0,0 +1,28 @@
+import fixLinks from 'src/freeze-dry/fix-links'
+
+describe('fixLinks tests', () => {
+    test('should append a base with the href of doc', async () => {
```

Comment:
This test is failing because of the change made by Gerben at [Commit](https://github.com/WebMemex/webmemex-extension/commit/3739e1f3ba58c1b4350b060bd56511c14b7346fd)